src/elexmodel/client.py

```diff
@@ -239,16 +248,21 @@ def get_estimates(
             aggregates, prediction_intervals, reporting_units, nonreporting_units, unexpected_units
         )
 
+        self.conformalization_data_unit_dict = {alpha: {} for alpha in prediction_intervals}
+        self.conformalization_data_agg_dict = {alpha: {} for alpha in prediction_intervals}
         for estimand in estimands:
             unit_predictions = model.get_unit_predictions(reporting_units, nonreporting_units, estimand)
             results_handler.add_unit_predictions(estimand, unit_predictions)
             # gets prediciton intervals for each alpha
-            alpha_to_unit_prediction_intervals = {
-                alpha: model.get_unit_prediction_intervals(
+            alpha_to_unit_prediction_intervals = {}
+            for alpha in prediction_intervals:
+                alpha_to_unit_prediction_intervals[alpha] = model.get_unit_prediction_intervals(
                     results_handler.reporting_units, results_handler.nonreporting_units, alpha, estimand
                 )
-                for alpha in prediction_intervals
-            }
+                self.conformalization_data_unit_dict[alpha][estimand] = (
+                    None if pi_method == "nonparametric" else model.get_conformalization_data_unit()
```

Comment:
this should probably work in the nonparametric case also

Reply:
done

Reply:
In this case now we don't need to set the none here in the nonparametric case, right?

Reply:
correct, changed